haxe/ui/core/Component.hx

```diff
@@ -1045,7 +1053,10 @@ class Component extends ComponentBase implements IComponentBase implements IClon
 
     private var _layoutLocked:Bool = false;
     public function lockLayout(recursive:Bool = false) {
```

Comment:
If this is causing problems / complication i think we can safely remove it for simplicity and reimplement it later... there maybe not be anything using it, and if there is, its not "essential"